itwin-platform-access/imodels-access-backend/src/BackendIModelsAccess.ts

```diff
@@ -354,8 +354,13 @@ export class BackendIModelsAccess implements BackendHubAccess {
     if (locks.length === 0)
       return [];
 
-    const result: LockProps[] = ClientToPlatformAdapter.toLockProps(locks[0]);
-    return result;
+    let lockProperties: LockProps[] = [];
+
+    for (const locksPage of locks){
+      Array.prototype.push.apply(lockProperties, ClientToPlatformAdapter.toLockProps(locksPage));
```

Comment:
Can use locks.map(ClientToPlatformAdapter.toLockProps) instead. More simple and consistent with other methods

Reply:
Fixed with https://github.com/iTwin/imodels-clients/pull/181/commits/f8067c104e9b5093c2b512b8afd72ffbde389d3b